system76-firmware-dialog

```diff
@@ -584,6 +587,8 @@ def send_notification(data):
     GLib.timeout_add_seconds(NOTIFICATION_TIMEOUT, notification.close)
     mainloop.run()
 
+Gtk.Window.set_default_icon_name("system76-driver")
```

Comment:
I believe this is causing the low-resolution Dash Icon.

Reply:
Do you know which function should be used instead? `gtk_window_set_icon_list()` perhaps?